### ISSUE TITLE
GC Editor crashes on startup.

### ISSUE THREAD

**Issue by Aspergerman7082** [RebelliousKiwi04/Python-Eaw-tools#4]
So far, every time I've tried to launch the GC Editor I'm met with the message: "TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'". I'm by no means an expert with coding...please help!

**Comment by RebelliousKiwi04** [comment]
Check the log file that should be generated alongside the executable, likely a game code issue

**Comment by Aspergerman7082** [comment]
Hi Kiwi, sorry if it has taken a minute for me to respond. I was working today. Yeah, I checked the log file, and it ends at: "Collecting Sort Order for conquest Full_Galaxy_Medium_Rebel_Era_1". I did notice that in the 3.2 conquest files, the campaign sorting order line of code is no longer present. I'm not sure if that is a part of the new GC map loading mechanism for TR or if I've somehow deleted that line of code...which is very possible, considering I'm self taught, and relatively new still to coding...but either way, thanks for taking the time to reply to my previous comment!
